pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -145,15 +150,15 @@ def load(fnames, tag=None, sat_id=None, malformed_index=False):
     # create altitude 'profile' at each location
     num = len(data['uts'])
     data['profiles'] = \
-        (('time', 'altitude'),
+        (('time', 'altitude_profile'),
```

Comment:
Is this whole bit needed?  Altitude is constant.

Reply:
These are for the profile data, which becomes an issue for xarray.  The data profiles simulate some sort of remote instrument (think COSMIC) which provides data as a function of altitude from a fixed-altitude platform.  I'm up for a different name here, but it needs to be unique so xarray will work.